### PR TITLE
clarify rollout behavior in deployment

### PR DESCRIPTION
**Pull request by chenxuc**
fix: #28884

### CODE REVIEW

File: content/en/docs/concepts/workloads/controllers/deployment.md [hunk 1]
```diff
@@ -759,7 +760,7 @@ apply multiple fixes in between pausing and resuming without triggering unnecess
     REVISION  CHANGE-CAUSE
     1   <none>
     ```
-* Get the rollout status to ensure that the Deployment is updated successfully:
+* Get the rollout status to ensure that the existing ReplicaSet is not changed:
```
Comment (sftim) [inline content/en/docs/concepts/workloads/controllers/deployment.md:763]:
```suggestion
* Get the rollout status to verify that the existing ReplicaSet has not changed:
```
?

**Review by sftim — COMMENTED** [review]
We could emphasize that whatever name the ReplicaSet had just before the new image was set, that's the name that its ReplicaSet still has (whilst it's paused) - and that eventually, after unpausing, the control plane cleans up the old, now-unused ReplicaSet.

**Comment by sftim** [comment]
/lgtm
as-is, this is an improvement

File: content/en/docs/concepts/workloads/controllers/deployment.md (hunk 1 above)
Comment (chenxuc) [reply content/en/docs/concepts/workloads/controllers/deployment.md]: Thanks for the rewording, I've updated this sentence.

**Pull request merged**